_ingest-pipelines/processors/dissect.md

```diff
@@ -0,0 +1,526 @@
+{
+  "docs": [
+    {
+      "_index": "testindex1",
+      "_id": "1",
+      "_source": {
+        "message": "New York, NY 10017"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+The substrings are appended to the `address` field, as shown in the following response:
+
+```json
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "testindex1",
+        "_id": "1",
+        "_source": {
+          "address": "New York|NY|10017",
+          "message": "New York, NY 10017"
+        },
+        "_ingest": {
+          "timestamp": "2024-01-22T22:30:54.516284637Z"
+        }
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+### Append with order modifier (`+` and `/n`)
+
+The append with order modifier combines the values of two or more keys into a single output value based on the order specified after the `/`. You have the flexibility to customize the separator that separates the appended values. The append modifier is useful for compiling multiple fields into a single formatted output line, constructing structured lists of data items, and consolidating values from various sources.  
+
+#### Example of usage
+
+The following example pipeline uses the append with order modifier to reverse the pattern order defined in the preceding pipeline. This pipeline specifies a separator to insert between the appended fields. If you don't specify a separator, all values will be appended together without a separator.
+
+```json
+PUT /_ingest/pipeline/dissect-test
+{
+  "description": "Pipeline that dissects web server logs",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{+address/3}, %{+address/2} %{+address/1}",
+        "append_separator": "|"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+You can test the pipeline using the following example:
+
+```json
+POST _ingest/pipeline/dissect-test/_simulate
+{
+  "docs": [
+    {
+      "_index": "testindex1",
+      "_id": "1",
+      "_source": {
+        "message": "New York, NY 10017"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+The substrings are appended into the `address` field in reverse order, as shown in the following response:
+
+```json
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "testindex1",
+        "_id": "1",
+        "_source": {
+          "address": "10017|NY|New York",
+          "message": "New York, NY 10017"
+        },
+        "_ingest": {
+          "timestamp": "2024-01-22T22:38:24.305974178Z"
+        }
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+### Named skip key
+
+The named skip key modifier excludes specific matches from the final output by using an empty key `{}` or `?` modifier within the pattern. For example, the following patterns are equivalent: `%{firstName} %{lastName} %{?ignore}` and `%{firstName} %{lastName} %{}`. The named skip key modifier is useful for excluding irrelevant or unnecessary fields from the output.
```

Comment:
This modifier name is also very awkward without some formatting to indicate that it's a name.